routes/users.js

```diff
@@ -27,7 +29,7 @@ router.post("/", async (req, res) => {
     const salt = await bcrypt.genSalt(10);
     user.password = await bcrypt.hash(user.password, salt);
     await user.save();
-    const token = jwt.sign({ _id: user._id }, config.get("PrivateKey")); //jwt encrypts {_id: user._id} using PrivateKey
+    const token = jwt.sign({ _id: user._id }, process.env.privateKey); //jwt encrypts {_id: user._id} using PrivateKey
```

Comment:
the auth.js file also has config.get("PrivateKey")

Reply:
change that one too

Reply:
oh yeah sounds good il change that rn